charts/backbeat/templates/replication/consumer_deployment.yaml

```diff
@@ -11,6 +11,8 @@ spec:
   replicas: {{ .Values.replication.consumer.replicaCount }}
   template:
     metadata:
+      annotations:
```

Comment:
It doesn't seem that this should be here if orbit isn't enabled. Should prob add a conditional for this.